engine/src/foundation-legacy.h

```diff
@@ -151,6 +151,7 @@ int UnicodeToUTF8(const uint16_t *lpSrcStr, int cchSrc, char *lpDestStr, int cch
 
 ////////////////////////////////////////////////////////////////////////////////
 
+bool MCStringConvertLineEndingsFromLiveCode(MCStringRef p_input, bool p_use_LF_line_endings, bool p_use_CR_line_endings, MCStringRef& r_output);
 bool MCStringConvertLineEndingsFromLiveCode(MCStringRef input, MCStringRef& r_output);
```

Comment:
As mentioned - I wonder if we need any of these functions anywhere - I *think* they can all be replaced with MCStringNormalizeLineEndings after the extra bool parameter controlling lsps conversion is added.